classes/class-p4-campaign-importer.php

```diff
@@ -192,5 +193,19 @@ public function set_imported_campaigns_as_drafts( $postdata, $post ) {
 
 			return $postdata;
 		}
+
+		/**
+		 * Needed to remove the underscore to expose the field in the API.
+		 * Use the value with underscore for old exports if the new field isn't present.
+		 *
+		 * @param array $post_meta The to be imported post meta fields.
+		 *
+		 * @return array The normalized post meta fields.
+		 */
+		public function read_old_campaign_template_attribute( $post_meta ) {
+			$post_meta['theme'] = $post_meta['theme'] ?? $post_meta['_campaign_page_template'];
```

Comment:
This caused a bunch of notices (`Undefined index: _campaign_page_template`) while importing. Probably not shown in production. 🤔